ci_config.json

```diff
@@ -14,7 +14,7 @@
   "broken_windows": [
     "arduinocore-avr",
     "google-woff2",
-    "libgpiod",
+    "libgpiod", "lz4",
```

Comment:
shouldn't be here. Rather a separate lz4 entry should be created and build_on windows should be false. See https://github.com/mesonbuild/wrapdb/blob/master/ci_config.json#L413

Reply:
The upstream software is intended to work on Windows, but instead it is broken -- which means that broken_windows is correct, and build_on is incorrect...

Reply:
`build_on` (+ rebase on master) seem to have worked.

@eli-schwartz What would I need to do to get past the [CI](https://github.com/jcaesar/wrapdb/runs/8135452483?check_suite_focus=true) with that?

Reply:
Make it actually work!!!!

Good grief.

Reply:
> Good grief.

Good joke. I haven't done any development on windows in more than a decade and I have no idea whether that's a problem with the CI (I don't really know what your CI does, after all…) which is why I was asking.

Thankfully, somebody else took care of it. I suppose I'll stop submitting PRs..

Reply:
What mostly annoys me about this is not that you tried to use an option someone told you to use. What annoys me is that someone told you to use it.

That being said, if the build files come from upstream, and "The upstream software is intended to work on Windows, but instead it is broken"... my instinctive assumption would be the first thing to try is to report an upstream bug.

> and I have no idea whether that's a problem with the CI (I don't really know what your CI does, after all…) which is why I was asking

And I answered that the problem with the CI indicates the software is broken.

But @neheb thinks that software being broken means that PRs should be merged anyway, so idk. Clearly there's a difference of opinion here. Maybe I'm a fool after all.

Reply:
> I have no idea whether that's a problem with the CI

It's a legitimate problem that the CI *allows* people to do things that aren't good per policy, and for that I apologize. We can, in fact, do better.

I updated the CI harness: I have no idea whether that's a problem with the CI

Now, the CI will actually tell you if something is okay or not.